front/src/applications/stdcm/components/StdcmWarningBox.tsx

```diff
@@ -25,8 +27,16 @@ const StdcmWarningBox = ({
   removeDestinationArrivalTime,
 }: StdcmWarningBoxProps) => {
   const { t } = useTranslation('stdcm');
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (errorType && ref.current) {
+      ref.current.scrollIntoView({ behavior: 'smooth' });
+    }
+  }, [errorType, errorDetails]);
```

Comment:
I think `errorType` should be enough. `errorDetails` is only defined when `errorType` goes to `bothPointAreScheduled` so the scroll will happen anyway.
```suggestion
  }, [errorType]);
```

Reply:
I wasn’t fully sure how `errorType` and `errorDetails` interact. I thought there might be cases where the same `errorType` happens with different `errorDetails` between renders.

Right now, based on how the form works, it’s true that `errorDetails` cannot change without `errorType` going to a different state first. However, I included it to handle any future changes or edge cases where this might happen.

If we’re confident this won’t be needed, I can simplify the dependency array and rely only on `errorType`. Let me know what you think.

Reply:
I find it weird to add a dependency in the deps array that isn't actually a dependency (not use in the useEffect).
It doesn't seem to be a use case to have the same `errorType` with different `errorDetails`. Even if others errors come in the future with an `errorDetails`, this shouldn't affect the `useEffect` since the `errorType` will always change.
Is it ok for you ?